### PR TITLE
Make all tests and lint pass, as it should be. Peace.

### PR DESCRIPTION
**Pull request by bakert**

### CODE REVIEW

File: decksite/templates/editmatches.mustache [hunk 1]
```diff
@@ -7,6 +7,9 @@
                     <td>
                         {{display_date}}
                     </td>
+                    <td>
+                        {{mtgo_id}}
```
Comment (silasary) [inline decksite/templates/editmatches.mustache]: This is a duplicate of lines 27-32

**Review by silasary — CHANGES_REQUESTED** [review]

File: decksite/templates/editmatches.mustache (hunk 1 above)
Comment (bakert) [reply decksite/templates/editmatches.mustache]: Oh, I guess we already did this :D

Comment (silasary) [reply decksite/templates/editmatches.mustache]: The ticket was that that column didn't actually work properly 😜

**Comment by silasary** [comment]
Don't revert it entirely!  The SQL addtion is still needed.

**Review by silasary — APPROVED** [review]

**Pull request merged**